src/main/java/frc/robot/Robot.java

```diff
@@ -59,34 +68,58 @@ public void teleopInit(){
-   Drive.arcadeDrive((-1 * Drive.getTurnPower()) * .2, Drive.getForwardPower() * .35, Shift);
-   CurrentPool.currentPool();
+    Drive.arcadeDrive(Drive.getTurnPower(), Drive.getForwardPower(), Shift);
+    Drive.Climber();
+      // if (Drive.driveStick.getRawButton(1)) {
+      //   Drive.R1.set(.5);
+      // } else if (Drive.driveStick.getRawButton(2)) {
+      //   Drive.R2.set(.5);
+      // } else if (Drive.driveStick.getRawButton(3)) {
+      //   Drive.L1.set(-.5);
+      // } else if (Drive.driveStick.getRawButton(4)) {
+      //   Drive.L2.set(-.5);
+      // } else {
+      //   Drive.R1.set(0);
+      //   Drive.R2.set(0);
+      //   Drive.L1.set(0);
+      //   Drive.L2.set(0);
+      // }
+  //  CurrentPool.currentPool();
    //System.out.println("R1: " + (Drive.R1.getEncoder().getPosition() / 10.75));
    //System.out.println("L1: " + (Drive.L1.getEncoder().getPosition() / 10.75));
    //Ballmech.ballMechTeleop();
     
     //recording mode
-    if (Drive.driveStick.getRawButton(4) == true){
-      changesInAngle = Drive.getNavxAngle() - startingAngle;
-      changesInRotations = (Drive.getRightPosition() + Drive.getLeftPosition() /2) - startingRotations;
-      //find changes in angles and rotations
-      //changeAngle = currentAngle - startingAngle
-      //changeDistance = currentRotations - startingRotations
-    } else if(Drive.driveStick.getRawButtonReleased(4)){
-      i++;
-      changeAngle[i] = changesInAngle;
-      changeRotations[i] = changesInRotations;
-      System.out.println("Angle Changes: " + changeAngle);
-      System.out.println("Rotation Chnages: " + changeRotations);
-      //put changes into the array 
-    } else if(Drive.driveStick.getRawButton(4) == false){
-      //keep finding starting positions and angles
-      startingAngle = Drive.getNavxAngle();
-      startingRotations = (Drive.getRightPosition() + Drive.getLeftPosition() /2);
+    // if (Drive.driveStick.getRawButton(4) == true){
+    //   changesInAngle = Drive.getNavxAngle() - startingAngle;
+    //   changesInRotations = (Drive.getRightPosition() + Drive.getLeftPosition() /2) - startingRotations;
+    //   //find changes in angles and rotations
+    //   //changeAngle = currentAngle - startingAngle
+    //   //changeDistance = currentRotations - startingRotations
+    // } else if(Drive.driveStick.getRawButtonReleased(4)){
+    //   i++;
+    //   changeAngle[i] = changesInAngle;
+    //   changeRotations[i] = changesInRotations;
+    //   System.out.println("Angle Changes: " + changeAngle);
+    //   System.out.println("Rotation Chnages: " + changeRotations);
+    //   //put changes into the array 
+    // } else if(Drive.driveStick.getRawButton(4) == false){
+    //   //keep finding starting positions and angles
+    //   startingAngle = Drive.getNavxAngle();
+    //   startingRotations = (Drive.getRightPosition() + Drive.getLeftPosition() /2);
+    // }
```

Comment:
Again, if this is not working, should we remove it and fix it in another pull request? Is it critical to remove this? 

I'm not sure of the value of the code if it is commented...